gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/AbstractRegionMap.java

```diff
@@ -1067,7 +1071,11 @@ public final RegionEntry updateRecoveredEntry(Object key, RegionEntry re,
           if (re.isTombstone()) {
             owner.scheduleTombstone(re, re.getVersionStamp().asVersionTag());
           }
-          owner.updateSizeOnPut(key, oldSize, owner.calculateRegionEntryValueSize(re));
+          int newSize = owner.calculateRegionEntryValueSize(re);
+          owner.updateSizeOnPut(key, oldSize, newSize);
+          // If owner is a local region i.e during backup account for entry overhead and value here.
```

Comment:
In what case is the owner non-null? -- LR and DR both? Here I don't mean base class local region but a region whose scope is "local". If the above is tru you might want to modify the comment a bit.

Reply:
ok.